sources/pmwebapi.go

```diff
@@ -137,6 +137,17 @@ func newPcpSource() (PcpSource, error) {
 			log.Fatal("NewRequest: ", err)
 		}
 
+		unit := ""
+		if pcpMetrics.Units != "" {
+			unit = "_" + pcpMetrics.Units
+		}
+		if strings.ToUpper(pcpMetrics.Type) == "COUNTER" {
+			pcpMetrics.Name = pcpMetrics.Name + unit + "_total"
```

Comment:
Probably also want to make sure you don't run into casing issues with "COUNTER", just to be safe. I think we've handled that elsewhere in the code already, so you should have an example to go off of.